### ISSUE TITLE
out of 6 test images, 4 images, instance segmentation result not there only black image

### ISSUE THREAD

**Issue by him009verma** [MaybeShewill-CV/lanenet-lane-detection#396]
in my trained model the total loss was near to 4.5 and the accuracy was 85%, why it is the case that the instance segmented images is appearing as black.

I used 220 images in training data for 2500 epochs(started training from snapshot with ur model), please help with the reason of above issue and also let me know the sample size of ur data. @MaybeShewill-CV please help

**Comment by MaybeShewill-CV** [comment]
@him009verma The sample size used to train the model was [256, 512]. You may adjust the dbscan threshold to see if that helps. Visualizing the feature embedding layer will also help you with it. To visualize the feature embedding layer you may refer
https://github.com/MaybeShewill-CV/lanenet-lane-detection/blob/9154c1497291276f03fb70a33149b152aa74cda3/tools/test_lanenet.py#L122-L124

**Comment by him009verma** [comment]
@MaybeShewill-CV thanks for the inputs will try these, sample size i mean the no. of images in the dataset.?, i want to know how much data is sufficient for this architecture to perform well as i have less manpower for the annotation task

**Comment by MaybeShewill-CV** [comment]
@him009verma Can not tell the accurate sample counts for a well performed model which is task dependent. Make annotation data as much as you can:)

**Comment by him009verma** [comment]
@MaybeShewill-CV hey I came across  a file name tusimple_ipm_remap.yml, can u please tell the how its created, i can see it has matrices of 640*640

**Comment by MaybeShewill-CV** [comment]
@him009verma Check this https://github.com/MaybeShewill-CV/Easy-Ipm-Client . It's used to convert front-view image into bird view:)

**Comment by him009verma** [comment]
@MaybeShewill-CV thanks ..!.. thats really helpfull, my data set is consisting the images where i already have the images in bird eye view ( i am using drone images) **so still using this IPM will help or not,** i have never used this so right now not in a situation to decide, your suggestion will help a lot...

**Comment by MaybeShewill-CV** [comment]
@him009verma ipm will not offer you more help in this situation:)

**Comment by monilshah1988** [comment]
> @MaybeShewill-CV thanks for the inputs will try these, sample size i mean the no. of images in the dataset.?, i want to know how much data is sufficient for this architecture to perform well as i have less manpower for the annotation task

How did you generate the binary image and instance images from the own dataset?
